Modelica/Mechanics/MultiBody/Parts.mo

```diff
@@ -2613,7 +2601,7 @@ November 3-4, 2003, pp. 149-158
         group="if animation = true",
         enable=animation));
     parameter Real hollowFraction=0.8
-      "1.0: Completely hollow, 0.0: rigid cylinder" annotation (Dialog(
+      "Outer radius / inner radius, i.e. 1.0: completely hollow, 0.0: rigid cylinder" annotation (Dialog(
```

Comment:
There is no outer radius or innder radius defined, but only wheel radius.

Reply:
Sorry, my mistake. Will fix it.